AndroidP/app/src/main/java/com/novoda/androidp/pt651_display_cutout/CutoutActivity.kt

```diff
@@ -0,0 +1,97 @@
+package com.novoda.androidp.pt651_display_cutout
```

Comment:
Why this weird package name? What does it mean?

Reply:
`pt651` is the task id in our Jira for this. I seemed like a good idea at that point :/

Reply:
I couldn't have ever understood it, like ever XD Can we switch to something more meaningful? :D external people don't even have access to our JIRA, and this is public

Reply:
Yep I'll drop the `pt651` entirely 👍